src/app-layout/__integ__/app-layout-focus-delegation.test.ts

```diff
@@ -52,17 +52,20 @@ describe.each(['classic', 'visual-refresh', 'visual-refresh-toolbar'] as const)(
         )
       );
 
-      //TODO create separate split panel test for appLayoutWidget is true
-      testIf(theme !== 'visual-refresh-toolbar')(
+      test(
         'split panel focus moves to slider on open and open button on close',
         setupTest(
           async page => {
-            await page.click(wrapper.findSplitPanel().findOpenButton().toSelector());
+            const splitPanelOpenActionEl =
+              theme === 'visual-refresh-toolbar'
+                ? wrapper.find(`button[data-testid="awsui-app-layout-trigger-slide-panel"]`).toSelector()
```

Comment:
Let's introduce a function on this test suite level to get this button selector based on the provided theme

```typescript
const findSplitPanelOpenButton(theme: string) => {
  // todo
}
```